src/unitxt/processors.py

```diff
@@ -277,3 +279,57 @@ def process_value(self, text: Any) -> Any:
 
         except:
             return 0
+
+
+class InferDictsToBinaryLogprobs(FieldOperator):
+    binary_class_names: list[str]
```

Comment:
If the code supports only two classes, let's call it pos_class_name, and neg_class_name.  
Otherwise, you need to add validation (e.g. the list has only two elements).

Another validation is to check that the two texts don't overlap ("Ok","Not Ok")

Reply:
done